src/test/scala/uk/gov/hmrc/api/service/UknwAuthCheckerApiService.scala

```diff
@@ -46,6 +46,22 @@ class UknwAuthCheckerApiService extends HttpClient {
       individualPayload
     )
 
+  def authorisationsdawd(
```

Comment:
Has this method been added in error? I can't see it used anywhere.

Reply:
Removed